src/components/collapsible-panel.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react';
+import { caretDownIcon, caretRightIcon } from '@jupyterlab/ui-components';
+
+export interface ICollapsiblePanelProps {
+  title: JSX.Element | string;
+  content: JSX.Element | string;
+  expanded?: boolean;
+}
+
+export function CollapsiblePanel(props: ICollapsiblePanelProps): JSX.Element {
+  const [expanded, setExpanded] = useState(props.expanded ?? false);
+
+  const handleChange = () => {
+    setExpanded(!expanded);
+  };
+
+  return (
+    <div className={'jp-collapsible-panel' + (expanded ? ' expanded' : '')}>
+      <div
+        className={
+          'jp-collapsible-panel-summary' + (expanded ? ' expanded' : '')
+        }
+      >
+        <div onClick={handleChange} className="jp-collapsible-panel-button">
+          {expanded ? (
+            <caretDownIcon.react container={document.createElement('SPAN')} />
+          ) : (
+            <caretRightIcon.react container={document.createElement('SPAN')} />
```

Comment:
Is the container property required here?

Reply:
used tag="span" instead of container property as done in other places where [.react is used in jupyterlab codebase](https://github.com/search?q=org%3Ajupyterlab+Icon.react&type=code). Serves the same purpose of providing container to icon. If not provided, defaults to <div>

Reply:
Figured out how to remove a container